src/views/AgmLocationChg.vue

```diff
@@ -90,14 +100,15 @@
                 </p>
               </header>
 
-              <div :class="{ 'invalid-section': !agmYearValid && showErrors }">
```

Comment:
I don't believe we need here a red border for everything. I don't see this type of behavior in other local filings. I think the red border should be to what's invalid.